neurodsp/tests/sim/test_aperiodic.py

```diff
@@ -33,6 +36,30 @@ def test_sim_powerlaw():
     sig = sim_powerlaw(N_SECONDS, FS, f_range=(2, None))
     check_sim_output(sig)
 
+def test_sim_frac_gaussian_noise():
+
+    # Simulate white noise. Do not normalize.
+    np.random.seed(0)
```

Comment:
I think setting a seed can be removed. If the tests fails because of random variation maybe a wider tolerance can be used.